test/functional/apps/management/_scripted_fields.js

```diff
@@ -56,6 +57,20 @@ export default function ({ getService, getPageObjects }) {
       await PageObjects.settings.removeIndexPattern();
     });
 
+    it('should not allow saving of invalid scripts', async function () {
+      await PageObjects.settings.navigateTo();
+      await PageObjects.settings.clickKibanaIndices();
+      await PageObjects.settings.clickScriptedFieldsTab();
+      await PageObjects.settings.clickAddScriptedField();
+      await PageObjects.settings.setScriptedFieldName('doomedScriptedField');
+      await PageObjects.settings.setScriptedFieldScript(`doc['iHaveNoClosingTick].value`);
+      await PageObjects.settings.clickSaveScriptedField();
+      await retry.try(async () => {
+        const formErrorExists = await find.existsByDisplayedByCssSelector('.euiFormErrorText');
```

Comment:
Then we can find the error with the test subject selector instead of the className.